src/index.ts

```diff
@@ -52,7 +52,9 @@ export function sass(opts: d.PluginOptions = {}): d.Plugin {
               results.code = `/**  sass error${err && err.message ? ': ' + err.message : ''}  **/`;
               resolve(results);
             } else {
-              results.dependencies = Array.from(sassResult.stats.includedFiles);
+              results.dependencies = Array.from(sassResult.stats.includedFiles as string[]).map((dep) =>
```

Comment:
@christian-bromann It looks like this is causing tests to fail on both Unix-based machines & Windows. The error message isn't super helpful, but it looks like it's these two tests that are to blame:
- [transform, import scss](https://github.com/ionic-team/stencil-sass/blob/9eecf2a2acda07f8ccb2df03486990ace3cf90a9/test/build.spec.ts#L42)
- [transform, import sass](https://github.com/ionic-team/stencil-sass/blob/9eecf2a2acda07f8ccb2df03486990ace3cf90a9/test/build.spec.ts#L98)

Both of which import a fixture file, `test-b` (either sass or scss, based on the test).

`test-b.sass` - https://github.com/ionic-team/stencil-sass/blob/main/test/fixtures/sass/test-b.sass
`test-b.scss` - https://github.com/ionic-team/stencil-sass/blob/main/test/fixtures/scss/test-b.scss

Both of which have an `@import` statement, which would align with the change we made here. LMK if there's anything I can do to help with this!

Reply:
I enhanced the mock for build context we have in the test. They all seem to pass now.